main.cpp

```diff
@@ -129,7 +129,7 @@ struct Animal
 
     int toeCreator()
     {
-        for(int i=0; i<numOfLegs; i++)
+        for( int i=0; i<numOfLegs; ++i )
```

Comment:
SPACE AROUND THE OPERATORS.  `=` is an operator. `<` is an operator.
- [ ] revise